timescaledb-tune.yaml

```diff
@@ -25,15 +25,28 @@ update:
     use-tag: true
     strip-prefix: v
 
+subpackages:
+  - name: timescaledb-tune-compat
+    description: "Compatibility subpackage for upstream docker entrypoint scripts"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}/usr/local/bin"
+          ln -s /usr/bin/timescaledb-tune "${{targets.contextdir}}/usr/local/bin/timescaledb-tune"
```

Comment:
We should create this symlink in the package that contains the entrypoint script since it depends on it being present anyway (unless this is useful somewhere else)